test/functional/cursor_tests.js

```diff
@@ -280,6 +281,37 @@ describe('Cursor', function() {
     }
   });
 
+  it('shouldCorrectlyExecuteCursorCountWithSecondaryReadPreference', {
```

Comment:
This old form of test naming came from setting exports on a module, and isn't a requirement anymore. For the sake of readability, I'd prefer to introduce spaces here.